app/assets/stylesheets/views/mod-actions.scss

```diff
@@ -224,6 +224,21 @@
     }
   }
 
+  .adjustment-restriction__container--text {
```

Comment:
Some of these could not be added as utility classes, hence added them to our css.